suricata/update/main.py

```diff
@@ -381,6 +381,11 @@ def fetch(self, url):
                     "will use latest cached version: %s", url, err)
                 return self.extract_files(tmp_filename)
             raise err
+        except IOError as err:
+            sys.stdout.write("\n")
+            logger.error("Failed to copy file from %s to %s: %s",
+                         tmp_fileobj.name, tmp_filename, err)
```

Comment:
I know I told you to add the filenames but this is getting too long. :( Maybe let's keep the message as ("Failed to copy file: ", err)?

Reply:
Hmm. I think the same.